CHANGELOG.md

```diff
@@ -1,5 +1,108 @@
 # Changelog
 
+## v1.11.0 (03.02.2021)
+
+### Enhancements
+
+#### :gear: Control Request Timeouts ([PR #5692](https://github.com/DevExpress/testcafe/pull/5692))
+
+New CLI options allow you to control the timeout for requests. If a request is not met with a response within the specified period, TestCafe throws an error. The CLI options are:
+
+* [--ajax-request-timeout](./docs/articles/documentation/reference/command-line-interface.md#--ajax-request-timeout-ms) controls the delay for fetch/XHR requests timeout
+* [--page-request-timeout](./docs/articles/documentation/reference/command-line-interface.md#--page-request-timeout-ms) sets the timeout for receiving webpages.
+
+Read the documentation for the corresponding configuration properties:
+
+* [ajaxRequestTimeout](./docs/articles/documentation/reference/configuration-file.md#ajaxrequesttimeout)
+* [pageRequestTimeout](./docs/articles/documentation/reference/configuration-file.md#ajaxrequesttimeout).
+
+The options are available in the [runner.run Method](./docs/articles/documentation/reference/testcafe-api/runner/run.md).
```

Comment:
This whole piece is a little weird.

First, you begin by saying that `CLI` options are introduced (I wouldn't do that, because we add TestCafe options to all the interfaces).

Then, you introduce the information about config properties by saying "Read documentation...". I first thought it means: "To understand more about the aforementioned CLI options, read the documentation about the corresponding config properties".

I understand you don't want to write similar introductory phrases, but here you can go with just headings:

```
*CLI*

* options

example (!)

*Configuration file*

* options

example (!)

*JavaScript API*

* options

example (!)
```

Yes, I think it's better to add examples as well.

Reply:
`The options are available in the [runner.run Method](./docs/articles/documentation/reference/testcafe-api/runner/run.md).`

What options are you describing? For example, are you referring to options that are listed below?

Reply:
The same options that are described a couple of lines above (CLI and Configuration File options)

Reply:
Thanks for the clarification. Can we update it to one of the following?

`The above options are available in`...

or

`These options are available in`...